modules/yaml-schema.json

```diff
@@ -154,20 +152,43 @@
                                 ]
                             },
                             "licence": {
-                                "type": ["array", "string"],
+                                "anyOf": [
+                                    {
+                                        "type": "array",
+                                        "minItems": 2,
+                                        "items": {
+                                            "type": "string"
+                                        },
+                                        "uniqueItems": true
+                                    },
+                                    {
+                                        "type": "string",
+                                        "pattern": "^[^\\[\\]]+$"
+                                    }
+                                ],
                                 "description": "Licence of the tool",
                                 "items": {
                                     "type": "string"
                                 },
-                                "uniqueItems": true
+                                "uniqueItems": true,
+                                "minItems": 2,
+                                "message": "Licence must either be a string, e.g. \"MIT\" or have at least 2 items otherwise"
```

Comment:
```suggestion
                                "message": "Licence must either be a string, e.g. \"MIT\" or be an array with at least 2 items otherwise"
```